interface/main/calendar/find_group_popup.php

```diff
@@ -0,0 +1,307 @@
+<?php
+/* Copyright (C) 2005-2007 Rod Roark <rod@sunsetsystems.com>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ */
```

Comment:
This needs a standard header and since you are creating this new script it, add yourself as the copyright and author.